### DISCUSSION TITLE
Is it possible to assign a dagster asset group to a dbt asset?

### DISCUSSION THREAD

**Discussion by jordunar** [dagster-io/dagster#18851] [Q&A] [answered]
Hello, I have a doubt related to groups and dbt assets. I have a dbt asset created with this code:

```
# execute dbt run with models/twins_base/ models
@dbt_assets(
    manifest=dbt_manifest_path, select="tag:twins_base"
)  # , dagster_dbt_translator=dagster_dbt_translator
def dbt_run_twins_base__twins_image_wf(
    context: AssetExecutionContext, dbt: DbtCliResource
):
    yield from dbt.cli(["run"], context=context).stream()
```

What I want to know is if there is any way to assign all the models declared as dagster assets with this code from above to a dagster asset group, as done with "normal" assets. For example:

```
@asset(group_name="cereal_assets")
def nabisco_cereals():
    return [1, 2, 3]
```

Thanks in advance

**Comment by rexledesma** [comment, marked as answer]
Our documentation covers this use case: https://docs.dagster.io/integrations/dbt/reference#customizing-group-names.

Related: https://github.com/dagster-io/dagster/discussions/17830#discussioncomment-7514379.

**Reply by jordunar** [reply]
Yes, I saw it right after asking. Thanks a lot!
